### DISCUSSION TITLE
Gramine GSC - Container Networking

### DISCUSSION THREAD

**Discussion by tiagorvmartins** [gramineproject/gramine#1443] [Q&A] [answered]
Hello I have a question regarding Gramine Shielded Containers,

I managed to build and sign a container using gsc.

Now I am trying to reach another container from this graminized docker to another hostname using other container name, and I am getting the following error:

hostname resolving error (lookup another-container on [168.63.129.16:53](http://168.63.129.16:53/): no such host)

Question:

For communication between containers do both of them need to be build and signed using gsc?
I guess an gsc container can't reach a standard docker container?

In that case, if all need to be built and signed by gsc, can at least the "another-container" not use sgx at all? Because I don't have that requirement, can be a graminized docker container but without sgx support?

Went through the GSC docs and couldn't find answer to my question, 

Thank you!

**Comment by kailun-qin** [comment, marked as answer]
@tiagorvmartins Thanks for the question! Pls kindly see my comments below.

> For communication between containers do both of them need to be build and signed using gsc?

No, I don't think so.

> hostname resolving error (lookup another-container on [168.63.129.16:53](http://168.63.129.16:53/): no such host)

Do you have `sys.enable_extra_runtime_domain_names_conf = true` in your manifest? Pls see the [gsc doc](https://gramine.readthedocs.io/projects/gsc/en/latest/index.html#issues-with-hostname-and-dns) for details.

**Reply by dimakuv** [reply]
Yep, I second what Kailun said. I think the problem is in `sys.enable_extra_runtime_domain_names_conf = true`, which is currently not set in your manifest file.

**Reply by tiagorvmartins** [reply]
Thank you so much for the kind and fast reply, I totally miss this point on the docs, it was the very last one. 
All working now! Thank you!
Have a nice day!
